### ISSUE TITLE
Installation Error - ModuleNotFoundError: No module named 'twisted'

### ISSUE THREAD

**Issue by pwprosap** [City-Bureau/city-scrapers#613]
Full error text when trying to run a test scraper after installing pipenv:
```Traceback (most recent call last):
  File "C:\Users\pwpro\Anaconda3\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\pwpro\Anaconda3\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\pwpro\.virtualenvs\city-scrapers-z7mfjnzw\Scripts\scrapy.exe\__main__.py", line 5, in <module>
  File "c:\users\pwpro\.virtualenvs\city-scrapers-z7mfjnzw\lib\site-packages\scrapy\__init__.py", line 27, in <module>
    from . import _monkeypatches
  File "c:\users\pwpro\.virtualenvs\city-scrapers-z7mfjnzw\lib\site-packages\scrapy\_monkeypatches.py", line 20, in <module>
    import twisted.persisted.styles  # NOQA
ModuleNotFoundError: No module named 'twisted'
```
In order to resolve this error, you will need to run the "pip run twisted" command from your folder where you cloned the Git repository into.

**Comment by pjsier** [comment]
Thanks for the report! Not sure I'm following the steps to reproduce, and I'm not able to replicate this locally with a fresh install using Python 3.5 and macOS.

Here are the commands I ran:
```
git clone https://github.com/City-Bureau/city-scrapers.git
cd city-scrapers
pipenv install --python 3.5
pipenv run scrapy crawl {SPIDER}
```

Did you see any output from `pipenv install` indicating any libraries failing to install?

**Comment by pwprosap** [comment]
I am installing on Windows not on a Mac. Currently working here in
Pittsburgh with Bonnie Fan. Unfortunately I had to restart my Git Bash
window so lost the potential error messages I ran into. I additionally had
to install win32api on my computer to finish things off.

**Comment by pjsier** [comment]
It sounds like Windows installation issues were addressed, and workarounds documented in bf1a2c442eb7371c72e62a9a6994d948da960391. Closing
